vue/sbc-common-components/src/components/BaseAddress.vue

```diff
@@ -316,76 +330,48 @@ export default class BaseAddress extends Mixins(ValidationMixin, CountriesProvin
   }
 
   /**
-   * Function to determine whether to use a country's known regions (ie, provinces/states).
-   * @param code The short code of the country.
-   * @returns Whether to use v-select or v-text-field for input.
+   * Determines whether to use a country's known regions (ie, provinces/states).
+   * @param code the short code of the country
+   * @returns whether to use v-select (true) or v-text-field (false) for input
    */
   private useCountryRegions (code: string): boolean {
     return (code === 'CA' || code === 'US')
   }
 
-  /**
-   * Enables AddressComplete for this instance of the address.
-   */
+  /** Enables AddressComplete for this instance of the address. */
   private enableAddressComplete (): void {
-    // If you want to use this component with the Canada Post AddressComplete service, it needs the following:
-    //  1. The AddressComplete JavaScript script include must be done to set up "window.pca".
-    //  2. Your AddressComplete account key must be defined as "window.addressCompleteKey".
+    // If you want to use this component with the Canada Post AddressComplete service:
+    // 1. The AddressComplete JavaScript script (and stylesheet) must be loaded.
+    // 2. Your AddressComplete account key must be defined.
     const pca = window['pca']
     const key = window['addressCompleteKey']
     if (!pca || !key) {
+      // eslint-disable-next-line no-console
+      console.log('AddressComplete not initialized due to missing script and/or key')
       return
     }
 
-    // Sets the id for the two form elements that are used by the AddressComplete code. If necessary this removes the
-    // id from previous elements.
-    this.moveElementId('street-address')
-    this.moveElementId('address-country-pca')
-
     // Destroy the old object if it exists, and create a new one.
     if (window['currentAddressComplete']) {
       window['currentAddressComplete'].destroy()
     }
     window['currentAddressComplete'] = this.createAddressComplete(pca, key)
   }
 
-  /**
-   * Sets the id attribute of the _named_ element to its name. If there was a pre-existing element with the id already
-   * set, it will be unset.
-   *
-   * @param name the name of the element for which to set the id.
-   */
-  private moveElementId (name: string): void {
-    const oldElement = document.getElementById(name)
-    const thisElement = this.$el.querySelector('[name="' + name + '"]')
```

Comment:
This only worked for the first instance of the BaseAddress component on a page, but there are many cases where there are multiple addresses on the same page.